queries.sql

```diff
@@ -93,3 +93,73 @@ select owners.full_name as full_name,count(*) as count from animals
 join owners on animals.owner_id=owners.id
 group by owners.full_name
 order by count DESC;
+
+
+/* Vet clinic database: add "join table" for visits */
+
+/* Who was the last animal seen by William Tatcher? */
+SELECT animals.name as animal_name,
+vets.name as vet_name, 
+visits.visit_date 
+FROM animals
+JOIN visits on visits.animal=animals.id
+JOIN vets on vets.id=visits.vet
+where vets.name like 'William Tatcher'
+ORDER BY visits.visit_date DESC LIMIT 1;
+
+/* How many different animals did Stephanie Mendez see? */
+select COUNT(DISTINCT animals.name) from animals
+JOIN visits ON animals.id=visits.animal
+JOIN vets ON visits.vet=vets.id
+where vets.name like 'Stephanie Mendez';
+
+/* List all vets and their specialties, including vets with no specialties. */
+SELECT * from vets 
+LEFT JOIN specialties ON vets.id=specialties.vet
+LEFT JOIN species ON species.id=specialties.species
```

Comment:
- [ ] If you update Stephanie Mendez's speciality, then you should have a result that includes her speciality as a pokemon vet. Kindly update the result in your pull request description. 👍